### PR TITLE
DTSERWFOUR-147-Alignment-Bug-fixes - Incorrect Spacing between Subhea…

### PR DESCRIPTION
**Pull request by pdamodarannair**
Incorrect Spacing between Subheader and content

### CODE REVIEW

**Review by shanavas123 — DISMISSED** [review]

**Review by rajkumar-balakrishnasamy — DISMISSED** [review]

**Review by vwagh-hw — APPROVED** [review]

File: receiptui/src/main/res/layout/fragment_tabbed_list_receipt.xml [hunk 1]
```diff
@@ -12,6 +12,7 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_alignParentTop="true"
+        android:layout_marginTop="16dp"
```
Comment (vwagh-hw) [inline receiptui/src/main/res/layout/fragment_tabbed_list_receipt.xml:15]: please define using `@dimen`

Comment (pdamodarannair) [reply receiptui/src/main/res/layout/fragment_tabbed_list_receipt.xml]: Updated

**Review by rajkumar-balakrishnasamy — APPROVED** [review]

File: transferui/src/main/res/layout/fragment_create_transfer.xml [hunk 2]
```diff
@@ -15,8 +15,10 @@
             android:id="@+id/create_transfer_container"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/section_header_height"
             android:orientation="vertical">
 
+
```
Comment (vwagh-hw) [inline transferui/src/main/res/layout/fragment_create_transfer.xml:21]: empty line

Comment (pdamodarannair) [reply transferui/src/main/res/layout/fragment_create_transfer.xml]: empty line removed

File: transfermethodui/src/main/res/layout/activity_select_transfer_method.xml [hunk 3]
```diff
@@ -22,7 +22,17 @@
...
+                app:popupTheme="@style/AppTheme.PopupOverlay" >
+
+                <TextView
+                    android:id="@+id/toolbar_title"
+                    style="@style/TextAppearance.AppCompat.Widget.ActionBar.Title"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_gravity="left"
+                    android:textColor="@android:color/white"
+                    android:textStyle="italic" />
```
[the remark this answers is not in the document]
Comment (pdamodarannair) [reply transfermethodui/src/main/res/layout/activity_select_transfer_method.xml:34]: Resolved

**Review by rajkumar-balakrishnasamy — APPROVED** [review]

**Review by shanavas123 — APPROVED** [review]

**Review by vwagh-hw — APPROVED** [review]

**Pull request merged**